### PR TITLE
Created utils.go

### PR DESCRIPTION
**Pull request by DarkLord017**
Please check all the functions

### CODE REVIEW

**Comment by DarkLord017** [comment]
Implemented consensus/utils.go & corrected consensus/rpc & consensus/consensus_core.go

Please check the functions in utils.go 
@star-gazer111

**Comment by DarkLord017** [comment]
@star-gazer111 sir consensus_core me kuch chize implemented nahi hai to usme changes me kru ya @0xpseudo kre ?
utsav ne bas us vakt jo kaam aa rha tha vahi implement kiya tha
PR abhi vapis draft krdi hai

File: consensus/utils.go [hunk 1]
```diff
@@ -1,15 +1,238 @@
...
+	Bellatrix ForkVersion
+	Capella   ForkVersion
+	Deneb     ForkVersion
+}
+
+type ForkVersion struct {
+	Epoch       uint64
+	ForkVersion [4]byte
+}
+
```
Comment (star-gazer111) [inline consensus/utils.go:28]: this stuff is already implemented [here](https://github.com/BlocSoc-iitr/selene/blob/dev/config/types.go). Why are you reimplementing the same stuff?

**Review by star-gazer111 — CHANGES_REQUESTED** [review]

File: consensus/utils.go (hunk 1 above)
Comment (DarkLord017) [reply consensus/utils.go:28]: Ok sir I'll correct it

**Comment by DarkLord017** [comment]
Please refer to PR #51

**Comment by star-gazer111** [comment]
@DarkLord017 this PR is for?

**Comment by DarkLord017** [comment]
It is already merged (it was for consensus/utils.go) refer #51

**Pull request closed without merging**